deploy/cluster-manager/config/rbac/cluster_role.yaml

```diff
@@ -32,6 +32,10 @@ rules:
 - apiGroups: ["apiextensions.k8s.io"]
   resources: ["customresourcedefinitions"]
   verbs: ["create", "get", "list", "update", "watch", "patch", "delete"]
+# Allow the registration-operator to update crds status
```

Comment:
May needs another PR in https://github.com/stolostron/backplane-operator/blob/main/pkg/templates/charts/toggle/cluster-manager/templates/cluster-manager-clusterrole.yaml to update the clusterrole.